src/cba-list-new/shadow.css

```diff
@@ -135,21 +141,24 @@ li .row:not([contenteditable="true"])
 
 li .row:not(.highlight):hover
 {
-  background: linear-gradient(var(--hover-primary) 0%,
+  /* background-color:  var(--color-selected); */
```

Comment:
Add hover effect to the top list items and add background color to selected items  as well discussed in the discourd.

Reply:
Ok,done